src/components/find-on-shelf/FindOnShelfPeriodicalDropdowns.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from "react";
+import { FC, useState } from "react";
+import ExpandMoreIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/ExpandMore.svg";
+import { HoldingsForBibliographicalRecordV3 } from "../../core/fbs/model";
+import makePeriodicalEditionsFromHoldings from "../material/helper";
+import { groupObjectArrayByProperty } from "../../core/utils/helpers/general";
+
+export interface FindOnShelfPeriodicalDropdownProps {
+  manifestationsHoldings: HoldingsForBibliographicalRecordV3[];
+  setSelectedPeriodical: (selectedWeek: string | null) => void;
+}
+
+const FindOnShelfPeriodicalDropdown: FC<FindOnShelfPeriodicalDropdownProps> = ({
+  manifestationsHoldings,
+  setSelectedPeriodical
+}) => {
+  const periodicalEditionsArray = makePeriodicalEditionsFromHoldings(
+    manifestationsHoldings[0].holdings
+  );
+  const sortedperiodicalEditionsArray = groupObjectArrayByProperty(
+    periodicalEditionsArray,
+    "volumeYear"
+  );
+
+  const [selectedYear, setSelectedYear] = useState<string>(
+    String(Object.keys(sortedperiodicalEditionsArray).sort().pop())
+  );
+
+  return (
+    <div className="modal-find-on-shelf__periodical-dropdowns">
+      <div className="dropdown dropdown--grey-borders">
+        <select
+          id="find-on-shelf-periodical-year"
```

Comment:
Consider whether this `id` value is needed.

An `id` value must be unique. I am not sure how this code guarantees this. I do not see if/how the id is needed either.

Reply:
Yeah, I'm not sure if I used enough time to fully understand this dropdown code before I reused it in find on shelf.
The ids are not needed as far as I can see, and I removed them now.